### DISCUSSION TITLE
Have Neo-tree hidden by default on Neovim start?

### DISCUSSION THREAD

**Discussion by dsincl12** [nvim-neo-tree/neo-tree.nvim#1325] [Q&A] [answered]
Hi everyone,

I'm trying to figure out how to have Neo-tree hidden when I start Neovim. If I open a familiar project or already know where I want to go there isn't really any point in having Neo-tree visible when I start Neovim so I would prefer to just open it when needed instead.

Any clues or suggestions?

**Comment by cseickel** [comment]
That is the normal behavior. Are you using a config like LazyVim or AstroVim?

**Reply by dsincl12** [reply]
Yup I'm using LazyVim

**Reply by cseickel** [reply]
You'll have to check their docs then, or ask on their project discussions/issues.

**Reply by dsincl12** [reply]
This is the default configuration in LazyVim:


```
opts = {
  sources = { "filesystem", "buffers", "git_status", "document_symbols" },
  open_files_do_not_replace_types = { "terminal", "Trouble", "trouble", "qf", "Outline" },
  filesystem = {
    bind_to_cwd = false,
    follow_current_file = { enabled = true },
    use_libuv_file_watcher = true,
  },
  window = {
    mappings = {
      ["<space>"] = "none",
      ["Y"] = function(state)
        local node = state.tree:get_node()
        local path = node:get_id()
        vim.fn.setreg("+", path, "c")
      end,
    },
  },
  default_component_configs = {
    indent = {
      with_expanders = true, -- if nil and file nesting is enabled, will enable expanders
      expander_collapsed = "",
      expander_expanded = "",
      expander_highlight = "NeoTreeExpander",
    },
  },
}
```

Am I missing something obvious? Because I can't see what could be the reason Neo-tree is open by default?

**Reply by pysan3** [reply]
Do you `nvim .`? 

https://github.com/LazyVim/LazyVim/blob/a50f92f7550fb6e9f21c0852e6cb190e6fcd50f5/lua/lazyvim/plugins/editor.lua#L46-L51

**Reply by dmshvetsov** [reply, marked as answer]
a quick solution for LazyVim can be

```lua
-- ./lua/plugins/neo-tree.lua or any other file where you define your plugins configuration
return {
  {
    "nvim-neo-tree/neo-tree.nvim",
    init = function()
      -- do nothing
    end,
  },
}
```

You can use `<leader>e` (default keymap) to toggle on/off the neo-tree explorer for files.

**Comment by Iamvismorf** [comment]
How do I do this in normal custom nvim?
